test/Microsoft.DotNet.Tools.Publish.Tests/Microsoft.DotNet.Tools.Publish.Tests.cs

```diff
@@ -31,13 +35,20 @@ public static IEnumerable<object[]> PublishOptions
             }
         }
 
+        public PublishTests() : base()
+        {
+            RepoRoot = Path.Combine(Directory.GetCurrentDirectory(), "..\\..\\");
```

Comment:
This is nasty. Why do we need this global.json thing now? Because we are not restoring? Even so, can we do anything better than ..\..\?